src/chapters/strings/string-methods.rst

```diff
@@ -0,0 +1,113 @@
+.. _string-methods:
+
+String Methods
+==============
+
+JavaScript provides many useful methods for string objects. Recall that a method is a function that "belongs to" a specific object. Methods will typically result in some operation being carried out on the data within an object. For strings, this means that our methods will typically transform the characters of the given string in some way.
+
+As we have learned, strings are immutable. Therefore, string methods will not change the value of a string itself, but instead will *return* a new string that is the result of the given operation.
+
+We saw this behavior in the ``toLowerCase`` example.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+   
+      let nonprofit = "LaunchCode";
+
+      console.log(nonprofit.toLowerCase());
+      console.log(nonprofit);
+
+   **Output**
+
+   ::
+
+      launchcode
+      LaunchCode
+
+While ``nonprofit.toLowerCase()`` evaluated to ``"launchcode"``, the value of ``nonprofit`` was left unchanged. This will be case for each of the string methods.
+
+Common String Methods
+---------------------
+
+.. index::
+   single: string; methods
+
+Here we present the most commonly-used string methods. You can find documentation for other string methods at:
+
+- `W3Schools <https://www.w3schools.com/jsref/jsref_obj_string.asp>`_
+- `MDN <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#Methods_2>`_
+
+
+.. list-table:: Common String Methods
+   :header-rows: 1
+
+   * - Method
+     - Syntax
+     - Description
+   * - :ref:`indexOf <string-indexof-examples>`
+     - ``stringName.indexOf(character)``
```

Comment:
``indexOf`` can search for strings as well as characters, so "Hello World !".indexOf("World") would return 6.